hack/update-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Could you add a header comment block explaining why this script exists, describe what's the problem it's trying to solve?